docs/setup/kubeedge_troubleshoot.md

```diff
@@ -0,0 +1,34 @@
+# Troubleshooting KubeEdge
+
+## Container keeps pending/ terminating
+
+1. Check the output of `kubectl get nodes`
+2. Check the output of `kubectl describe pod <your-pod>`
+3. Check the `edgecore` logs for any errors.
+4. Check the architecture of the node running `edgecore` and make sure that container image you are trying to run is of the same architecture.
+   For example, if you are running `edgecore` on Raspberry Pi 4, which is of `arm64v8` architecture, the nginx image to be executed would be `arm64v8/nginx` from the docker hub.
+
+5. Also, check that the `podSandboxImage` is correctly set as defined in [Modification in edgecore.yaml](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/kubeedge_configure.md#modification-in-edgecoreyaml).
+
+6. If all of the above is correctly set, login manually to your edge node and run your docker image manually by
+
+  ```shell
+   docker run <your-container-image>  ```
```

Comment:
need one line break here, otherwise it won't be rendered correctly, https://github.com/kubeedge/kubeedge/blob/5732c90aa2bf6cbe3e7b89d6882a75115f7f72d6/docs/setup/kubeedge_troubleshoot.md

Reply:
Thanks @chendave , I will fix that.